packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -2093,6 +2096,46 @@ export async function putDefaultRole(
   });
 }
 
+export async function putGetStartedChecklistItem(
+  req: AuthRequest<{
+    item: GetStartedChecklistItem;
+    type: keyof GetStartedChecklist;
+  }>,
+  res: Response
+) {
+  const context = getContextFromReq(req);
+  const { org } = context;
+  const { item, type } = req.body;
+
+  const emptyChecklist: GetStartedChecklist = {
+    features: [],
+    experiments: [],
+    importedExperiments: [],
+  };
+
+  const newChecklist = {
+    getStartedChecklists: {
+      ...(org.getStartedChecklists || emptyChecklist),
+    },
+  };
+
+  const index = newChecklist.getStartedChecklists[type].findIndex(
+    (c) => c.step === item.step
+  );
+  if (index === -1) {
+    newChecklist.getStartedChecklists[type].push({
+      step: item.step,
+      isCompleted: true,
+    });
+  }
+
```

Comment:
All API endpoints that write to Mongo should have permission checks.  Ideally the check would be dependent on which step is being checked.  If someone doesn't have permission to create environments, they shouldn't be able to mark that step as completed.